### DISCUSSION TITLE
WordKeys in Adoption Contract Template

### DISCUSSION THREAD

**Discussion by anual0928** [sheltermanager/asm3#1568] [Q&A / Help]
I am new to ASM so likely have some very naive questions but I can't seem to find the answers anywhere. I am having issues with finding/using the right wordkeys for our contract template and the manual doesn't seem to have enough detail to help. Is there anywhere else to look before I ask here? I have looked through the Q&A here and tried to look through the archived ones as well.

**Comment by Tetzor** [comment]
The wordkey section of the manual is the best place to find the information on how to construct these, the qualifier section is particularly important to single out a particular record from many. eg. VaccinationGivenRecentRabies - date of most recently given vaccination of type "Rabies".

Jon

**Reply by anual0928** [reply]
I am having issues with the owner wordkey.  On the adoption contract, when a couple adopts, I use <<ownername>> and it comes up nicely as "John and Jane Smith".  However, for signatures, I want to have their names separate so I can get both signatures.  I need to have "John Smith" in one section and "Jane Smith" in another.  I have tried using "1" and "2" after OwnerName but that doesn't do it.  Please advise.

**Comment by Tetzor** [comment]
Hi, OwnerName retrieves the records display name, so in the case of couples it will display "John & Jane Smith".  To display couples names separately you would need to use  FirstNames1 LastName1 FirstNames2 LastName2. 

It's only possible to have one signature field in a document template, so if using a single document for a couple and they both want to sign, they would need to fit both signatures into the single signature field. 

Jon

**Comment by anual0928** [comment]
Hi Jon,

I finally got it to work but I had to use FirstNames LastName FirstNames2
and LastName2.  I could not use "1" for the first person in the couple.  So
that problem is solved. And thanks for the tip on the signature field.  At
the moment, we are not using the system to have people electronically sign.
Hopefully we will get there one day.

Now I have an issue with the brand name of the microchip.  I am using the
wordkey <<MicrochipManufacturer>>.  The one I just tried, the brand is
EIDAP yet on the document, it populates as Trovan.  Your help would be
greatly appreciated again.

Thanks,
Launa

**Comment by Tetzor** [comment]
Hi, I think EIDAP are a registry rather than a manufacturer, and Trovan is a manufacturer they use from what I can find online. We use the prefix at the start of the number eg. 956 to identify them.

Jon
